src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -267,6 +267,96 @@ console.log("Duration: " + script_duration)
 
     </Step>
 
+    <Step>
+    ## Configure and develop a scripted API test [#scripted-api-test]
+    Using scripted API monitors, you can build complex monitoring workflows with Node.js and the `got` module. For example, you can authenticate with an API, handle errors based on the response code, and parse the response body. To do this, you'll:
+
+    1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic Monitoring**</DoNotTranslate>.
+    2. Click the <DoNotTranslate>**Create monitor**</DoNotTranslate> button.
+    3. Enter the name and details of your monitor. For example, `sitename.com`.
+    4. Select the locations from where you want your monitor to run. For example, Mumbai, Seoul, Columbus, or Montreal.
+    5. Choose a frequency for how often your monitor will run on each location selected. For example, `5 mins`.
+    6. Set a notification method to alert your team when performance incidents occur.
+    7. You are now ready to write your script. See this example script to test making an API request and processing the response.
+
+```js
+/**
+ * Script Name: Best Practices - Node 16.10.0
```

Comment:
This example script is too complex. It includes built in retries and a retry delay. This could be a useful example to include in our quickstarts, but is unnecessarily complex for a best practices API example. 

Any best practice example should just include typical interactions with the $http object and nothing more.